src/batch/src/executor/join/lookup_join.rs

```diff
@@ -169,7 +169,14 @@ impl<C: BatchTaskContext> ProbeSideSource<C> {
 
         let prost_exchange_source = ProstExchangeSource {
             task_output_id: Some(TaskOutputId {
-                task_id: Some(ProstTaskId::default()),
+                task_id: Some(ProstTaskId {
+                    // FIXME: We should replace this random generated uuid to current query_id for
```

Comment:
This is executed using `execute` rpc call, so plan id is not that important and we don't need to show it in dashboard.

Reply:
OK. `execute` rpc call will stil need the taskoutput channel so I think this pr is good enough.